### PR TITLE
V2

### PR DESCRIPTION
**Pull request by Tagadda**
## Problem

- v1

## Solution

- v2 !!

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+[integration]
+yunohost = ">= 11.1"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = true
+disk = "50M" # FIXME: replace with an **estimate** minimum disk requirement. e.g. 20M, 400M, 1G, ...
+ram.build = "50M" # FIXME: replace with an **estimate** minimum ram requirement. e.g. 50M, 400M, 1G, ...
+ram.runtime = "50M" # FIXME: replace with an **estimate** minimum ram requirement. e.g. 50M, 400M, 1G, ...
```
Comment (Tagadda) [inline manifest.toml:27]:
```suggestion
disk = "1G"
ram.build = "2500M"
ram.runtime = "700M"
```

**Comment by lapineige** [comment]
I can't use the resolve conflicts button for some reason 🤔
Can you do it @Tagadda ?

**Comment by Tagadda** [comment]
Neither do I... Let's manually merge the thing ^^

**Comment by lapineige** [comment]
For the rest of it, I can't really review it… if CI is happy, manual testing is still required but it should be fine…

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
!testme

File: conf/.env.production.sample [hunk 2]
```diff
@@ -254,31 +254,31 @@ LDAP_TLS_NO_VERIFY=true
 # ---------------
  
 # Maximum allowed character count
-MAX_TOOT_CHARS=__MAX_TOOT_CHARS__
+MAX_TOOT_CHARS=500
```
Comment (lapineige) [inline conf/.env.production.sample:257]: Why coming back to 5000 while Glitch allows a lot more (by default) ? :(

File: conf/.env.production.sample [hunk 3]
```diff
@@ -254,31 +254,31 @@ LDAP_TLS_NO_VERIFY=true
...
-MAX_PROFILE_FIELDS=__MAX_PROFILE_FIELDS__
+MAX_PROFILE_FIELDS=4
 
 # Maximum allowed display name characters
-MAX_DISPLAY_NAME_CHARS=__MAX_DISPLAY_NAME_CHARS__
+MAX_DISPLAY_NAME_CHARS=30
 
 # Maximum allowed poll options
-MAX_POLL_OPTIONS=__MAX_POLL_OPTIONS__
+MAX_POLL_OPTIONS=5
```
Comment (lapineige) [inline conf/.env.production.sample:272]: Same as previous, why so few ?

File: conf/.env.production.sample [hunk 4]
```diff
@@ -254,31 +254,31 @@ LDAP_TLS_NO_VERIFY=true
...
 
 # Maximum allowed poll option characters
-MAX_POLL_OPTION_CHARS=__MAX_POLL_OPTION_CHARS__
+MAX_POLL_OPTION_CHARS=100
```
Comment (lapineige) [inline conf/.env.production.sample:275]: Same. Couldn't have a little more space to breathe ? :)

Comment (Tagadda) [reply conf/.env.production.sample:275]: Those are the defaults and you're supposed to change it from the config panel (which is broken right now)

**Comment by Tagadda** [comment]
!testme

File: conf/.env.production.sample (hunk 4 above)
Comment (lapineige) [reply conf/.env.production.sample:275]: But why stick to such strict defaults while we can offer more comfort *by default*, out-of-the-box ?
Especially if right now it's not easy to adjust this as its config panel is broken.

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
!testme

**Comment by Tagadda** [comment]
!tagadaci-bookworm ?

**Pull request closed without merging**